pkg/network/deviceinfo/deviceinfo_test.go

```diff
@@ -0,0 +1,165 @@
+/*
+* This file is part of the KubeVirt project
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* Copyright the KubeVirt Authors.
+*
+ */
+package deviceinfo_test
+
+import (
+	"fmt"
+
+	networkv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	virtv1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/pkg/libvmi"
+	"kubevirt.io/kubevirt/pkg/network/deviceinfo"
+)
+
+var _ = Describe("DeviceInfo", func() {
+
+	const (
+		booHashedIfaceName  = "pod6446d58d6df"
+		fooHashedIfaceName  = "pod2c26b46b68f"
+		deviceInfoPlugin    = "deviceinfo"
+		nonDeviceInfoPlugin = "non_deviceinfo"
+	)
+
+	networkStatusWithMixedNetworksFmt := `[
```

Comment:
Can this be converted into a function that accepts the arguments and returns the result? It will be cleaner than using `Sprintf` inline.

Reply:
It is not a format anymore.